redis/connection.py

```diff
@@ -526,8 +527,10 @@ def __init__(
     ):
         """
         Initialize a new Connection.
-        To specify a retry policy, first set `retry_on_timeout` to `True`
-        then set `retry` to a valid `Retry` object
+        To specify a retry policy for specific errors, first set
```

Comment:
Given this, I think retry_on_error should start as an empty list, rather than None. Rather than change the type (from None), it provides a hint to the future user, at least based on the signature.

Reply:
changed